nyx_client/nyx_client/client.py

```diff
@@ -386,6 +388,7 @@ def _data_from_response_object(self, obj: dict[str, Any]) -> Data:
             genre=obj["genre"],
             size=obj["size"],
             custom_metadata=(Property.from_dict(prop) for prop in obj.get("customMetadata", ())),
+            connection_id=obj.get("connectionId", "default"),
```

Comment:
Since "nothing" is returned if it doesn't exist - I think it'd be better to just have `None` (we haven't agreed anything about "default" being a special keyword and it doesn't say anything in the API)